### PR TITLE
Fix BTC address calculation - Closes #3772

### LINKED ISSUE #3772: Fix BTC address calculation
### Expected behavior
We should calculated all addresses correctly.

### Actual behavior
The BTC address is calculated wrong.

### Which version(s) does this affect? (Environment, OS, etc...)
2.0.0

### PR DESCRIPTION
**Pull request by reyraa**
### What was the problem?
This PR resolves #3772

### How was it solved?
Since we added an API response normalizer to the BTC API, we don't need to distinguish the responses in the utility level.

### How was it tested?
unit tests

### CODE REVIEW

File: app/dev-app-update.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
-owner: LiskHQ
+owner: reyraa
```
Comment (ManuGowda) [inline app/dev-app-update.yml:1]: should this be LiskHQ?

File: app/src/modules/autoUpdater.js [hunk 2]
```diff
@@ -12,17 +13,17 @@ const getErrorMessage = (error) => {
 export default ({ // eslint-disable-line max-statements
   autoUpdater, dialog, win, electron,
 }) => {
+  autoUpdater.logger = logger;
```
Comment (ManuGowda) [inline app/src/modules/autoUpdater.js:16]: maybe remove the auto updater changes from this PR?

File: app/dev-app-update.yml (hunk 1 above)
Comment (reyraa) [reply app/dev-app-update.yml:1]: For sure. But shouldn't we resolve it in your PR?

Comment (ManuGowda) [reply app/dev-app-update.yml:1]: yes, i think we should just keep only BTC changes in this PR

File: app/src/modules/autoUpdater.js (hunk 2 above)
Comment (reyraa) [reply app/src/modules/autoUpdater.js]: Removed. It was a problem with the branch base.

**Review by ManuGowda — COMMENTED** [review]
@reyraa the change itself LGTM, although I must say I do not have the full context of codebase, but the changes itself looks good. 

I see you have added in PR `unit tested`, I do not see any unit test, could you please add the tests?

**Review by isalga — APPROVED** [review]

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**